netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -0,0 +1,215 @@
+        {
+            var targetPathToItem = new Dictionary<string, ITaskItem>(StringComparer.OrdinalIgnoreCase);
+            foreach (var item in items)
+            {
+                var targetPath = GetTargetPath(item);
+
+                ITaskItem existingItem;
+
+                if (targetPathToItem.TryGetValue(targetPath, out existingItem))
+                {
+                    // a conflict was found, determine the winner.
+                    var winner = HandleConflict(existingItem, item);
+
+                    if (winner != existingItem)
+                    {
+                        // replace existing item and add it to conflict list.
+                        targetPathToItem[targetPath] = item;
+                        conflicts.Add(existingItem);
+                    }
+                    else
+                    {
+                        // no need to replace item, just add new item to conflict list.
+                        conflicts.Add(item);
+                    }
+                }
+                else
+                {
+                    targetPathToItem[targetPath] = item;
+                }
+            }
+        }
+
+
+        private static string GetSourcePath(ITaskItem item)
+        {
+            var sourcePath = item.GetMetadata("HintPath");
+
+            if (String.IsNullOrWhiteSpace(sourcePath))
+            {
+                sourcePath = item.ItemSpec;
+            }
+
+            return sourcePath;
+        }
+
+        static readonly string[] s_targetPathMetadata = new[] { "TargetPath", "Path" };
+        private static string GetTargetPath(ITaskItem item)
+        {
+            foreach (var metadata in s_targetPathMetadata)
+            {
+                var targetPath = item.GetMetadata("TargetPath");
+
+                if (!String.IsNullOrWhiteSpace(targetPath))
+                {
+                    return targetPath.Replace('\\', '/');
+                }
+            }
+
+            var sourcePath = GetSourcePath(item);
+
+            return Path.GetFileName(sourcePath);
+        }
+
+
+        private ITaskItem HandleConflict(ITaskItem item1, ITaskItem item2)
+        {
+            var conflictMessage = $"Encountered conflict between {item1.ItemSpec} and {item2.ItemSpec}.";
+
+            var sourcePath1 = GetSourcePath(item1);
+            var sourcePath2 = GetSourcePath(item2);
+
+            var exists1 = File.Exists(sourcePath1);
+            var exists2 = File.Exists(sourcePath2);
+
+            if (!exists1 && !exists2)
+            {
+                Log.LogMessage($"{conflictMessage}.  Could not determine winner because {sourcePath1} and {sourcePath2} do not exist.  Arbitrarily choosing {item1.ItemSpec}.");
+                return item1;
+            }
+
+            if (exists1 && !exists2)
+            {
+                Log.LogMessage($"{conflictMessage}.  Choosing {item1.ItemSpec} because {sourcePath1} exists and {sourcePath2} does not.");
```

Comment:
Meh, I don't think we should be the point of failure here.  I'll just let if flow through.